projects/components/card/src/testing/card.harness.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseHarnessFilters, ContentContainerComponentHarness, HarnessPredicate, TestElement } from '@angular/cdk/testing';
+import { PsHeaderHarness } from '@prosoft/components/header/src/testing/header.harness';
+
+export interface PsCardHarnessFilters extends BaseHarnessFilters {
+  caption?: string | RegExp;
+  description?: string | RegExp;
+}
+
+export const enum PsCardSection {
+  FOOTER = '.mat-card-footer',
+  ACTIONS = '.mat-card-actions',
+}
+
+export class PsCardHarness extends ContentContainerComponentHarness<PsCardSection> {
+  static hostSelector = 'ps-card';
+
+  private _header = this.locatorForOptional(PsHeaderHarness);
+  private _actionsChildren = this.locatorForAll(PsCardSection.ACTIONS + ' > *');
+  private _footerChildren = this.locatorForAll(PsCardSection.FOOTER + ' > *');
+
+  static with(options: PsCardHarnessFilters = {}): HarnessPredicate<PsCardHarness> {
+    return new HarnessPredicate(PsCardHarness, options)
+      .addOption('caption', options.caption, (harness, title) => HarnessPredicate.stringMatches(harness.getCaptionText(), title))
+      .addOption('description', options.description, (harness, subtitle) =>
+        HarnessPredicate.stringMatches(harness.getDescriptionText(), subtitle)
+      );
+  }
+
+  async getCaptionText(): Promise<string> {
```

Comment:
maybe only a getHeader method, then you dont have to forward all header harness methods